extensions/amp-consent/0.1/consent-state-manager.js

```diff
@@ -379,7 +382,8 @@ export class ConsentInstance {
         })
       ).length;
 
-      if (size > CONSENT_STORAGE_MAX) {
+      // Size restriction only applies to documents servered from a cache.
+      if (this.viewer_.isEmbedded() && size > CONSENT_STORAGE_MAX) {
```

Comment:
Viewer.isEmbedded doesn't guarantee storageAPI support. I recommend adding a new method to Storage service and check that instead.